controllers/validation/account_validation_controller.go

```diff
@@ -439,6 +447,52 @@ func (r *AccountValidationReconciler) Reconcile(ctx context.Context, request ctr
 			}
 		}
 	}
+
+	// check if account belongs to accountpool
+	if !account.IsBYOC() {
+		awsRegion := config.GetDefaultRegion()
+		awsSetupClient, err := r.awsClientBuilder.GetClient(controllerName, r.Client, awsclient.NewAwsClientInput{
+			SecretName: utils.AwsSecretName,
+			NameSpace:  awsv1alpha1.AccountCrNamespace,
+			AwsRegion:  awsRegion,
+		})
+		if err != nil {
+			connErr := fmt.Sprintf("unable to connect to default region %s", awsRegion)
+			reqLogger.Error(err, connErr)
+			return ctrl.Result{}, err
+		}
+
+		if account.Spec.RegionalServiceQuotas != nil {
+			if account.Status.RegionalServiceQuotas == nil {
+				err = accountcontroller.SetCurrentAccountServiceQuotas(reqLogger, r.awsClientBuilder, awsSetupClient, &account, r.Client)
+				if err != nil {
+					reqLogger.Error(err, "failed to set account service quotas")
+					return reconcile.Result{}, err
+				}
+				err := r.statusUpdate(&account)
+				if err != nil {
+					return reconcile.Result{}, err
+				}
+
+				return reconcile.Result{RequeueAfter: 30 * time.Second}, nil
+			} else {
+				if account.HasOpenQuotaIncreaseRequests() {
+					switch utils.DetectDevMode {
+					case utils.DevModeProduction:
+						return accountcontroller.GetServiceQuotaRequest(reqLogger, r.awsClientBuilder, awsSetupClient, &account, r.Client)
+					}
+				}
+				for _, quotas := range account.Status.RegionalServiceQuotas {
```

Comment:
This is because in Dev mode quota request are not open on the AWS side. This is also to prevent the constant re-queue of the `account_validation_controller` after all the quota request are completed

Reply:
Ok, that makes sense. Thanks!